devops/ansible/site.yml

```diff
@@ -0,0 +1,134 @@
+- name: Setting up Girder
+  hosts: localhost
+  connection: local
+  vars:
+    required_ansible_version: 2.5.1
+  tasks:
+    - fail: msg="Playbook requires ansible >= {{required_ansible_version}}"
+      when: ansible_version.string is version_compare(required_ansible_version, '<')
+
+    - name: Set some facts
+      set_fact:
+        girder_host: "{{girder_host|default('localhost')}}"
+        girder_port: "{{girder_port|default(8080)}}"
+        girder_scheme: "http"
+
+    - name: Create Girder eSimMon admin user
+      girder:
+        host: "{{girder_host}}"
+        port: "{{girder_port}}"
+        scheme: "{{girder_scheme}}"
+        user:
+          firstName: "esimmonadmin"
+          lastName: "esimmonadmin"
+          login: "esimmonadmin"
+          password: "{{ admin_password }}"
+          email: "esimmonadmin@kitware.com"
+          admin: yes
+        state: present
+
+    - name: Create filesystem assetstore
+      girder:
+        host: "{{girder_host}}"
+        port: "{{girder_port}}"
+        scheme: "{{girder_scheme}}"
+        username: "esimmonadmin"
+        password: "{{ admin_password }}"
+        assetstore:
+          name: "Filesystem Assetstore"
+          type: "filesystem"
+          root: "/assetstore"
+          current: true
+        state: present
+      when: s3_bucket is not defined
```

Comment:
We can remove the when line, this was leftover from edp.

Reply:
Removed